react-stopwatch-component/src/index.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class StopWatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.state = { started: false };
+    this.seconds = 0;
```

Comment:
Since the number of seconds passed changes over time, it should be a property of state.